heartbeat/galera

```diff
@@ -488,6 +501,65 @@ detect_first_master()
     set_bootstrap_node $best_node
 }
 
+galera_status()
```

Comment:
mysqld may in fact be running (e.g. during SST) before it creates its pidfile. This new function adds check for that specific case.
It's implemented in the spirit of https://github.com/ClusterLabs/resource-agents/blob/master/rgmanager/src/resources/mysql.sh#L157